tracer/src/Datadog.Trace/LifetimeManager.cs

```diff
@@ -60,7 +60,7 @@ public static LifetimeManager Instance
             }
         }
 
-        public TimeSpan TaskTimeout { get; set; } = TimeSpan.FromSeconds(30);
+        public TimeSpan TaskTimeout { get; set; } = TimeSpan.FromSeconds(60);
```

Comment:
I had to look at the code to know what this is used for 😅 . Do you mind renaming `TaskTimeout` to clarify, maybe `ShutdownTaskTimeout`? It looks like there's only one usage in line 126 below, anyway. (Not sure why it has a public setter)